src/screens/AddParticipant/index.js

```diff
@@ -262,6 +262,6 @@ const styles = StyleSheet.create({
     marginTop: 16
   },
   subtitleStyle: (isIsset) => ({
-    color: isIsset ? COLORS.holyTosca : '#C4C4C4'
+    color: isIsset ? COLORS.holytosca : '#C4C4C4'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant `COLORS.holyTosca` has been renamed to `COLORS.holytosca`. Please ensure that this renaming is consistent across all files and references to avoid any undefined values.
```diff
- color: isIsset ? COLORS.holyTosca : '#C4C4C4'
+ color: isIsset ? COLORS.holytosca : '#C4C4C4'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->